nearuplib/nodelib.py

```diff
@@ -74,36 +99,13 @@ def check_and_setup(nodocker, binary_path, image, home_dir, init_flags, no_gas_p
             exit(1)
 
         if chain_id == 'devnet':
-            genesis_time = get_genesis_time(chain_id)
-            if genesis_time == genesis_config['genesis_time']:
-                print(f'Our genesis version up to date')
-            else:
-                print(
-                    f'Remote genesis time {genesis_time}, ours is {genesis_config["genesis_time"]}')
-                print(
-                    f'Update genesis config and remove stale data for {chain_id}')
-                os.remove(os.path.join(home_dir, 'genesis.json'))
-                download_genesis('devnet', home_dir)
-                if os.path.exists(os.path.join(home_dir, 'data')):
-                    shutil.rmtree(os.path.join(home_dir, 'data'))
+            if check_and_update_genesis(chain_id, home_dir):
+                # For devnet, also update config because boot nodes changed
                 print(f'Update devnet config for new boot nodes')
                 os.remove(os.path.join(home_dir, 'config.json'))
```

Comment:
Why are boot nodes updated only on devnet? Also it doesn't look like a very good idea to remove `config.json` just to update boot nodes, since it will delete all manual changes previously done.

Reply:
devnet configs is generated by `neard testnet`, you know that generate new 4 folders with new pub keys, therefore pubkey@ip:24567 changed. And yes it's suboptimal to override people's config file, however I think it's only overriding devnet's config file, it's updating so often to make sense its config subject to change with upstream server. For betanet and testnet it won't override people's config